src/types/db.ts

```diff
@@ -0,0 +1,4 @@
+// Result row used by TileDatabase to return BLOB mbtile data encoding feature boundaries
```

Comment:
I can still see myself being led astray by the reference to a row.
```suggestion
// Result row used by TileDatabase to return BLOB mbtile data encoding feature boundaries for a single tile.
```
or change the name to TileDataResult.